rich/console.py

```diff
@@ -1924,21 +1929,24 @@ def _check_buffer(self) -> None:
                     if WINDOWS:
                         try:
                             file_no = self.file.fileno()
+                            stdout_num = sys.stdout.fileno()
```

Comment:
These probably don't need to be in a try except blog. Suggest replacing them with constants. `pty.STDOUT_FILENO` and `pty.STDERR_FILENO` contain the values so they don't need to be hard coded.